pkg/action/cl002/ac001/crs.go

```diff
@@ -36,8 +36,8 @@ func newCRs(releases []v1alpha1.Release, host string) (v1alpha2.ClusterCRs, erro
 			Description:       explainerCommand,
 			Owner:             key.Organization,
 			Region:            key.RegionFromHost(host),
-			ReleaseComponents: p.Components().Latest(),
-			ReleaseVersion:    p.Version().Latest(),
+			ReleaseComponents: p.Components().Previous(),
+			ReleaseVersion:    p.Version().Previous(),
```

Comment:
Changing the cluster creation to use the previous patch version, because we are going to upgrade the cluster in a later action to the latest patch version.